### PR TITLE
Entity_Lecture4-Entity

### PR DESCRIPTION
**Pull request by wi9916**

### CODE REVIEW

**Comment by wi9916** [comment]
Start

File: Entity/EF.Domain/Models/Person.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        public string Name { get; set; }
+        public string ImageName { get; set; }
+        public string PhoneNumber { get; set; }
+        public string Email { get; set; }
+        public string About { get; set; }       
+        //public string TestFild { get; set; }        
+        //public Customer Customer { get; set; }
+
+        public List<Customer> Customers = new List<Customer>();
+        //public string Test { get; set; }
```
Comment (yevhenbilushko) [inline Entity/EF.Domain/Models/Person.cs]: remove all comments

File: Entity/Entity.Data/DataContext.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Entity.Data
+{
+    public class DataContext: DbContext
+    {
+        public DbSet<Person> Persons { get; set; }
```
Comment (yevhenbilushko) [inline Entity/Entity.Data/DataContext.cs:13]: 2 tabless in DB - seems like not enough
recheck it
maybe u should add more tables

**Review by yevhenbilushko — DISMISSED** [review]

**Comment by yevhenbilushko** [comment]
fix and merge

**Pull request closed without merging**